src/sentry/integrations/github_enterprise/webhook.py

```diff
@@ -115,61 +112,67 @@ def get_secret(self, event, host):
             return None
 
     def handle(self, request: Request) -> Response:
-        body = bytes(request.body)
-        if not body:
-            logger.warning("github_enterprise.webhook.missing-body", extra=self.get_logging_data())
-            return HttpResponse(status=400)
-
-        try:
-            handler = self.get_handler(request.META["HTTP_X_GITHUB_EVENT"])
-        except KeyError:
-            logger.warning("github_enterprise.webhook.missing-event", extra=self.get_logging_data())
-            logger.exception("Missing Github event in webhook.")
-            return HttpResponse(status=400)
-
-        if not handler:
-            return HttpResponse(status=204)
-
-        try:
-            event = json.loads(body.decode("utf-8"))
-        except json.JSONDecodeError:
-            logger.warning(
-                "github_enterprise.webhook.invalid-json",
-                extra=self.get_logging_data(),
-                exc_info=True,
-            )
-            logger.exception("Invalid JSON.")
-            return HttpResponse(status=400)
-
-        try:
-            host = request.META["HTTP_X_GITHUB_ENTERPRISE_HOST"]
-        except KeyError:
-            logger.warning("github_enterprise.webhook.missing-enterprise-host")
-            logger.exception("Missing enterprise host.")
-            return HttpResponse(status=400)
-
-        secret = self.get_secret(event, host)
-        if not secret:
-            logger.warning("github_enterprise.webhook.missing-integration", extra={"host": host})
-            return HttpResponse(status=400)
-
-        try:
-            # Attempt to validate the signature. Older versions of
-            # GitHub Enterprise do not send the signature so this is an optional step.
-            method, signature = request.META["HTTP_X_HUB_SIGNATURE"].split("=", 1)
-            if not self.is_valid_signature(method, body, secret, signature):
+        with configure_scope() as scope:
```

Comment:
Use Hide whitespace to make it easier to review this block since I'm doing a right shift.
<img width="179" alt="image" src="https://user-images.githubusercontent.com/44410/207946592-a938c31a-fdd8-4f67-8b7a-2e370c1a09ae.png">